lib/fieldhand/repository.rb

```diff
@@ -134,7 +135,7 @@ def get(identifier, arguments = {})
     private
 
     def paginator
-      @paginator ||= Paginator.new(uri, :logger => logger, :timeout => timeout, :bearer_token => bearer_token)
+      @paginator ||= Paginator.new(uri, :logger => logger, :timeout => timeout, :bearer_token => bearer_token, :headers => headers)
```

Comment:
As above, do we still need to send `bearer_token` here now that it will be incorporated into `headers` by this point?